### PR TITLE
Dt 808 one search field

### PR DESCRIPTION
**Pull request by VilleMiekkoja**

### CODE REVIEW

File: test/search.js [hunk 1]
```diff
@@ -1,45 +0,0 @@
-var suite = require('./api/suite.js').suite;
```
Comment (hannesj) [inline test/search.js]: Why remove tests entirely instead of adjusting them?

File: app/component/search/fake-search-with-button.cjsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+React       = require 'react'
+Icon        = require '../icon/icon'
+
+FakeSearchWithButton = (props) ->
+  <div className="search-form">
```
Comment (hannesj) [inline app/component/search/fake-search-with-button.cjsx]: Could we simplify the dom structure?

Comment (VilleMiekkoja) [inline app/component/search/fake-search-with-button.cjsx]: Managed to simplify it little

File: test/api/browser-upgrade.js [hunk 3]
```diff
@@ -72,32 +72,53 @@ module.exports = function (browser) {
...
       browser.click('#destination');
     },
 
-    enterText: browser.origin.enterText,
-
+    enterText: function type(text) {
+      browser.expect.element('#search-destination').to.be.enabled.before(ELEMENT_VISIBLE_TIMEOUT);
+      browser.pause(100);
+      browser.setValue('#search-destination', text);
+      browser.pause(1000); //wait for suggestions
```
Comment (hannesj) [inline test/api/browser-upgrade.js]: Couldn't this be done with something like `browser.expect.element('#suggestions').to.be.visible.before(ELEMENT_VISIBLE_TIMEOUT)`

File: app/component/search/search-main-container.cjsx [hunk 4]
```diff
@@ -191,6 +173,7 @@ class SearchTwoFieldsContainer extends React.Component
           className={"search-header__button" + if @state.selectedTab == "destination" then "--selected" else ""}
           label={destinationSearchTabLabel}
           value={"destination"}
+          id={"destination"}
```
Comment (hannesj) [inline app/component/search/search-main-container.cjsx]: Parenthesis not needed around strings

**Pull request merged**